spec/index.html

```diff
@@ -319,49 +319,50 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
-      <dfn class="lint-ignore">transparent</dfn>.</p>
-
+      in another triple. 
+      Being an [=RDF triple=], it denotes a <a>proposition</a>.
     <p>
       A <dfn>reifying triple</dfn> is a triple where the <a>predicate</a> is <code>rdf:reifies</code>
       and the <a>object</a> is a <a>triple term</a>.
       The <a>subject</a> of that triple is called a <dfn>reifier</dfn>, and it can be the
       subject or object of other triples.
-      When the triple term of a <a>reifying triple</a> also [=appears=] in the
-      same graph as an <a>asserted triple</a>, the subset of triples that share
-      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.
     </p>
-
     <p>
-      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]], may have shortcuts
-      for specifying <a>reifying triples</a> and <a>triple annotations</a> more
-      succinctly.
+      A [=reifier=] may denote a variety of things that are related to the triple term's [=proposition=], 
+      such as a statement or belief that the [=proposition=] holds.
+      It is expected that the [=reifiers=] (rather than the [=triple terms=]) will be used in further descriptions.
+      This section describes this common usage.
     </p>
 
-    <p>The following diagram represents a <a>reifying triple</a> of an unasserted, abstract <a>triple term</a>, and a <a>triple</a> relating the <a>reifier</a> to another resource.</p>
+    <p>
+      For example, the following diagram represents a <a>reifying triple</a> of a <a>triple term</a>, together with a <a>triple</a> that includes the <a>reifier</a> as the [=subject=].
+    The latter describes the reifer as a claim made by `:Bob` that the proposition denoted by the reified triple term holds. In other words, `:Bob` claims that `:Alice`'s name is "Alice".
+    </p>
 
     <figure id="fig-triple-term">
       <a href="triple-term.svg">
         <object data="triple-term.svg" aria-describedby="fig-triple-term-alt"
           aria-label="An RDF graph containing a triple that references an unasserted triple term (with grey dashed arc) via a reifier"></object>
       </a>
       <figcaption id="fig-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> that references an abstract <a>triple term</a> (which is unasserted, depicted using a grey, dashed arc) from a <a>reifier</a>; and a triple describing this reifier.
+        An <a>RDF graph</a> containing a <a>reifying triple</a> that references a <a>triple term</a> (which is unasserted, depicted using a grey, dashed arc) from a <a>reifier</a>; and a triple describing this reifier.
       </figcaption>
     </figure>
 
+    <p>
+      In this example, the [=proposition=] denoted by the [=triple term=] (<code>:Alice</code>'s name being <code>"Alice"</code>) is not asserted.
+      For the [=triple term=] to also be asserted, it must also [=appear=] in the same graph as an <a>asserted
+      triple</a>. 
```

Comment:
As @TallTed pointed out above, the sentence is not internally consistent, using qualifying once the "proposition" and once the "triple term" as "asserted". In our terminology, "triples" are asserted, so we need to fix the 1st mention.

Another issue is the use of the defined term "appears", which is incorrect. An asserted triple does not "appear" in a graph (in our definition), only the terms that constitute it.
```suggestion
      In this example, the [=proposition=] denoted by the [=triple term=] (<code>:Alice</code>'s name being <code>"Alice"</code>) is not claimed to be true.
      That would only be the case if the triple used as a triple term was also an [=asserted triple=] in the RDF graph.
```